src/ts/components/playbacktogglebutton.ts

```diff
@@ -17,8 +17,9 @@ export class PlaybackToggleButton extends ToggleButton<ToggleButtonConfig> {
 
     this.config = this.mergeConfig(config, {
       cssClass: 'ui-playbacktogglebutton',
-      text: i18n.getLocalizer('playPause'),
-      ariaLabel: i18n.getLocalizer('play'),
+      text: i18n.getLocalizer('play'),
+      onAriaLabel: i18n.getLocalizer('pause'),
+      offAriaLabel: i18n.getLocalizer('play'),
```

Comment:
We should also update `HugePlaybackToggleButton` accordingly - either replace `ariaLabel` with the two new properties, or just remove `ariaLabel` and rely on the `PlaybackToggleButton` super class.

Reply:
Done in 6aa57a2e